scripts/badger/bribe_ecosystems.py

```diff
@@ -104,30 +111,48 @@ def main(
             mantissa,  # uint256 amount
         )
 
-    if badger_bribe_in_balancer > 0:
-        prop = web3.solidityKeccak(["address"], [r.balancer.B_20_BTC_80_BADGER_GAUGE])
-        mantissa = int(badger_bribe_in_balancer * Decimal(1e18))
+    def bribe_balancer(gauge, mantissa):
+        prop = web3.solidityKeccak(["address"], [gauge])
+        mantissa = int(mantissa)
 
         badger.approve(bribe_vault, mantissa)
+        print(gauge, prop, mantissa)
```

Comment:
to facilitate the readability of the prop hash against https://hhand.xyz/proposal/balancer
```suggestion
        print(gauge, prop.hex(), mantissa)
```

Reply:
oh that was just for testing actually